Makefile

```
@@ -126,6 +126,17 @@ sepolia: compile-all
 test: compile-all
 	mix test --no-start --exclude spectest
 
+#### BEACON NODE OAPI ####
+OAPI_NAME = beacon-node-oapi
+OAPI_VERSION := $(shell cat .oapi_version)
+$(OAPI_NAME).json:
```

We should make this depend on `.oapi_version`:
```suggestion
$(OAPI_NAME).json: .oapi_version
```

That way changes in the version cause `make` to update the json. Note that this wasn't needed for spec-tests due to them having their version appended to the file name.